highcharts-shared-styles.html

```diff
@@ -1,11 +0,0 @@
-<link rel="import" href="../shadycss/apply-shim.html">
```

Comment:
Please use the @andre's suggestion from here: https://stackoverflow.com/questions/48281365/how-use-custom-style-or-shared-styles-in-polymer-3-preview. To leverage shared styles to avoid code-redundancy.